resources/schemas/dbscripts/postgresql/targetedms-22.002-22.003.sql

```diff
@@ -0,0 +1,35 @@
+
+-- Junction table between Run and AuditLogEntry
+CREATE TABLE targetedms.RunAuditLogEntry
+(
+    RowId                   SERIAL NOT NULL,
+    VersionId               BIGINT NOT NULL,
+    AuditLogEntryId         INT NOT NULL,
+    Container               entityId NOT NULL,
```

Comment:
Can we avoid storing a Container here directly? We've only added it as a direct column to targetedms tables when it was strictly required for perf reasons. Everywhere else, we join to the run table (sometimes through a series of FKs) to figure out the Container. This makes it easier to move runs between containers, for example.

Reply:
Ok I removed container from this table and operations on the table